src/tir/schedule/analysis/analysis.cc

```diff
@@ -890,7 +890,7 @@ IterVarType GetLoopIterType(const StmtSRef& loop_sref) {
           }
         });
       }
-      return false;
+      return true;
```

Comment:
What does this change do? Seems like it could cause side effects.

Reply:
This change makes `GetLoopIterType` recurse into sub-blocks (`true` means visit sub-nodes). This makes `GetLoopIterType` correct in the presence of nested blocks.